solutions/security/investigate/notes.md

```diff
@@ -64,8 +51,8 @@ Use the **Notes** page to view and interact with all existing notes. To access t
 * Filter notes by the user who created them or by the object they’re attached to (notes can be attached to alerts, events, or Timelines)
 * Examine the contents of a note (click the text in the **Note content** column)
 * Delete one or more notes
-* Examine the alert or event that a note is attached to (click the **Expand alert/event details** ![Preview alert or event action](../../../images/security-notes-page-document-details.png "") icon)
-* Open the Timeline that the note is attached to (click the **Open saved timeline** ![Open Timeline action](../../../images/security-notes-page-timeline-details.png "") icon)
+* Examine the alert or event that a note is attached to (click the **Expand alert/event details** ![Preview alert or event action](../../../images/security-notes-page-document-details.png "title =20x20") icon)
+* Open the Timeline that the note is attached to (click the **Open saved timeline** ![Open Timeline action](../../../images/security-notes-page-timeline-details.png "title =20x20") icon)
```

Comment:
Same thing with this icon image and the one on the previous line. My only guess is it's a bug with using the title attribute to define image size.

Reply:
The bug persists, even without the sizing attribute. :( Here's what it looks like after I removed the sizing attribute:

<img width="556" alt="Screenshot 2025-02-24 at 10 23 27 PM" src="https://github.com/user-attachments/assets/7bffa109-3f6b-4bcc-b01d-d6a9b723d644" />

The image size is slightly larger and still displays on a separate line.